circle.yml

```diff
@@ -0,0 +1,31 @@
+# circle.yml -- Font-Awesome
+
+machine:
+  environment:
+    DEPLOY_TOOLKIT: $HOME/deploy_toolkit
+    FAILURE_FILE: /tmp/build_failures.txt
+dependencies:
+  pre:
+    # Need deploy_toolkit before anything else
+    - git clone -b master git@github.com:loanlifecycle/deploy_toolkit.git $DEPLOY_TOOLKIT
+    # pycurl and paho-mqtt are used by build_notifications.py
+    - pip install pycurl paho-mqtt mock
+    - sudo apt-get install --assume-yes --force-yes ruby-dev nodejs npm build-essential
+    - sudo gem install bundler
+    - sudo npm install -g grunt grunt-cli less less-plugin-clean-css
+  override:
+    - bundle install
+    - npm install
+# Just stubbing this since there is no need for anything to run here.
+database:
+  pre:
+    - echo "Database:pre section is empty."
+  override:
+    - echo "Database:override section is empty."
+test:
+  pre:
+  override:
+    - bundle exec jekyll build || ( touch $FAILURE_FILE && exit 1 )
```

Comment:
Do we need to also deploy force framework with this submodule included? Similar to https://github.com/loanlifecycle/design-system/blob/develop/circle.yml